app/templates/events/eventList.html

```diff
@@ -205,8 +211,8 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
   <div class="tab-pane {{'active' if activeTab == 'bonnerScholarsEvents' else ''}} fade show" id="pills-bonner-scholars" role="tabpanel" aria-labelledby="pills-bonner-scholars-tab">
     {{createTable(bonnerEvents, "bonner")}}
   </div>
-  <div class="tab-pane {{'active' if activeTab == 'otherEvents' else ''}} fade show" id="pills-one-time" role="tabpanel" aria-labelledby="pills-one-time-tab">
-    {{createTable(otherEvents, "other")}}
+  <div class="tab-pane {{'active' if activeTab == 'celtsLabor' else ''}} fade show" id="pills-one-time" role="tabpanel" aria-labelledby="pills-one-time-tab">
+    {{createTable(celtsLabor, "other")}}
```

Comment:
Since we are getting rid of the "other" program column, we should probably give it a new name here and change all the references of "other" elsewhere